pkg/test/framework/components/environment/kube/flags.go

```diff
@@ -179,4 +214,8 @@ func init() {
 			"a given cluster appears in the 'istio.test.kube.config' flag. This topology also determines where control planes should "+
 			"be deployed. If not specified, the default is to deploy a control plane per cluster (i.e. `replicated control "+
 			"planes') and map every cluster to itself (e.g. 0:0,1:1,...).")
+	flag.StringVar(&networkTopology, "istio.test.kube.networkTopology",
```

Comment:
Just a thought... I wonder if it might be good to invert this (i.e map network name to a set of clusters). I'm not sure exactly what the syntax should be, however, since symbols start to get tricky and we don't want to interfere with flag processing. Maybe something like `networkA:0/1,networkB:2/3`? WDYT?

I'm fine leaving it as-is as well.